build.assets/Dockerfile-centos7-fips

```diff
@@ -132,14 +132,13 @@ ENV GOEXPERIMENT=boringcrypto \
 # Install node.
 RUN yum install -y python3
 ARG NODE_VERSION
-ENV NODE_URL="https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-${BUILDARCH}.tar.xz"
 ENV NODE_PATH="/usr/local/lib/nodejs-linux"
 ENV PATH="$PATH:${NODE_PATH}/bin"
 RUN export NODE_ARCH=$(if [ "$BUILDARCH" = "amd64" ]; then echo "x64"; else echo "arm64"; fi) && \
-     export NODE_URL="https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz" && \
-     mkdir -p ${NODE_PATH} && \
-     curl -o /tmp/nodejs.tar.xz -Lf ${NODE_URL} && \
-     tar -xJf /tmp/nodejs.tar.xz -C /usr/local/lib/nodejs-linux --strip-components=1
+    export NODE_URL="https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz" && \
+    mkdir -p ${NODE_PATH} && \
+    curl -o /tmp/nodejs.tar.xz -Lf ${NODE_URL} && \
```

Comment:
Quotes, curl flags:

```suggestion
    mkdir -p "$NODE_PATH" && \
    curl -o /tmp/nodejs.tar.xz -fsSL "$NODE_URL" && \
```

Reply:
I didn't want to change the quotes because PROTOC_TARBALL also doesn't use them, suggesting that this is Docker expanding the vars and not the shell.

Reply:
These are all env vars, so it's OK for us to rely on shell expansion. I would keep the quotes, but no need to backtrack just for this.